microgateway_decorator/lib/microgateway.js

```diff
@@ -85,7 +85,11 @@ var getMicroData = function(){
         },
         optional: {
             APIGEE_MICROGATEWAY_CUSTOM: process.env.APIGEE_MICROGATEWAY_CUSTOM ? parseCustom(process.env.APIGEE_MICROGATEWAY_CUSTOM) : {},
-            APIGEE_MICROGATEWAY_PROCESSES: process.env.APIGEE_MICROGATEWAY_PROCESSES ? process.env.APIGEE_MICROGATEWAY_PROCESSES : ""
+            APIGEE_MICROGATEWAY_PROCESSES: process.env.APIGEE_MICROGATEWAY_PROCESSES ? process.env.APIGEE_MICROGATEWAY_PROCESSES : "",
+            APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL: process.env.APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL ? process.env.APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL : false,
+            APIGEE_MICROGATEWAY_NODEJS_URL: process.env.APIGEE_MICROGATEWAY_NODEJS_URL ? process.env.APIGEE_MICROGATEWAY_NODEJS_URL : "",
+            APIGEE_MICROGATEWAY_NODEJS_FILENAME: process.env.APIGEE_MICROGATEWAY_NODEJS_FILENAME ? process.env.APIGEE_MICROGATEWAY_NODEJS_FILENAME : "",
+            APIGEE_MICROGATEWAY_NODEJS_VERSION: process.env.APIGEE_MICROGATEWAY_NODEJS_VERSION ? process.env.APIGEE_MICROGATEWAY_NODEJS_VERSION : ""
```

Comment:
We shouldn't need these items in here. If any of these are incorrect this file won't be run and we don't use these anywhere in the script execution.

Reply:
Yes, I'll remove those as well.